wherehows-web/app/components/datasets/containers/dataset-health.ts

```diff
@@ -137,6 +139,8 @@ export default class DatasetHealthContainer extends Component {
       severityMetrics: healthData.severity,
       tableData: healthData.detail
     });
+
+    return health; // Do something with health information
```

Comment:
API is actually not ready yet so was thinking of setting up the API handler but still keeping mock data in case people want to see it in staging still. Return statement is so that ts didn't yell at us...